launch/testnet/server.go

```diff
@@ -44,8 +44,11 @@ func Setup() *Server {
 	// creating the rpcbus
 	rpcBus := rpcbus.New()
 
-	m := mempool.NewMempool(eventBus, rpcBus, nil)
-	m.Run()
+	// Light nodes don't need to keep a mempool
+	if !cfg.Get().General.WalletOnly {
```

Comment:
Is `WalletOnly config` kept for backward compatibility? Wouldn't `General.LightNode` name make code more informative?

Reply:
True. Will adjust